homeassistant/components/amberelectric/sensor.py

```diff
@@ -43,6 +44,27 @@ def friendly_channel_type(channel_type: str) -> str:
     return "General"
 
 
+def interval_to_datum(
```

Comment:
We should not put all this information in the state attributes. Ideally the old data should be migrated to either new entities or using service responses to allow the user to view the data and do their own processing to it.